static/css/styles.css

```diff
@@ -1,18 +1,18 @@
 /* NumPy Color Palette */
-/* #FFC553 Mustard */
-/* #4DABCF Maximum Blue */
-/* #4D77CF Han Blue */
-/* #FFFFFF White */
-/* #EEEEEE Isabelline */
-/* #6C7A89 Aurometalsaurus */
-/* #013243 Warm Black */
+/* rgb(255, 197, 83)   */
+/* rgb(77, 171, 207)   */
+/* rgb(77, 119, 207)   */
+/* rgb(255, 255, 255)  */
+/* rgb(238, 238, 238)  */
+/* rgb(108, 122, 137)  */
+/* rgb(1, 50, 67)      */
```

Comment:
I personally think it's really nice to add a description here for each color-- I don't have any idea what each of these are, and it will be easier to use them in the future if we know which one is the blue one, for example.

Reply:
@joelachance It’s a fair point. The issue with the current color names is that some of them have been assigned to a different RGB since we selected the color palette for numpy.org. (For more context, refer to https://github.com/numpy/numpy.org/pull/405#issuecomment-798808702.) This is bound to cause confusion.

What do you think of naming the palette as follows:
rgb(255, 197, 83) NumPy Yellow (or NumPy Mustard)
rgb(77, 171, 207) NumPy Ndarray Blue
rgb(77, 119, 207) NumPy Deep Blue
rgb(255, 255, 255) White
rgb(238, 238, 238) NumPy Cloud Gray 
rgb(108, 122, 137) NumPy Slate Gray
rgb(1, 50, 67) NumPy Warm Black

@joelachance and everyone else, feel free to suggest your names for the color palette.

Reply:
I like them! I think any color description that's close to the color is 'good enough'. Developers will easily know what color they're using with any description.

Reply:
@joelachance Great! 
@aerikpawson Please incorporate the suggested above into your PR.